testutil/ciao-down/workload.go

```diff
@@ -75,7 +80,41 @@ func (wkld *workload) save(ws *workspace) error {
 	return nil
 }
 
-func loadWorkloadData(ws *workspace, workloadName string) ([]byte, error) {
+func loadWorkloadData(ctx context.Context, ws *workspace, workloadName string) ([]byte, error) {
+	var absPath string
+	u, err := url.Parse(workloadName)
+	if err == nil {
+		switch u.Scheme {
+		case "http", "https":
+			workloadFile, err := ioutil.TempFile("", ".workload")
+			if err != nil {
+				break
+			}
+
+			absPath = workloadFile.Name()
+			defer func() { _ = os.Remove(absPath) }()
+
+			// 60 seconds should be enough to download the workload file
+			ctx, cancelFunc := context.WithTimeout(ctx, 60*time.Second)
+			cancelFunc()
+			err = getFile(ctx, workloadName, workloadFile, downloadProgress)
+			if err != nil {
+				return nil, fmt.Errorf("Unable download workload file from %s: %v", workloadName, err)
+			}
+		case "file":
+			absPath = path.Join("/", u.Host, u.Path)
+		default:
+			absPath, _ = filepath.Abs(u.Path)
```

Comment:
Maybe we should just return an error here.  For example, if someone were to specify

 ftp://user:password@host:port/path 

as a URL we should not be trying to access an absolute path on the local machine.

Also, what does

url.Parse("ciao") return?  Does this error out?